setup.py

```diff
@@ -0,0 +1,11 @@
+from setuptools import setup
+
+setup(
+    name='libero schemas',
+    version='0.1',
+    description='libero xml api schemas',
+    url='http://github.com/libero/schemas',
+    author='eLife sciences publications, ltd.',
```

Comment:
```suggestion
    maintainer='eLife Sciences Publications, Ltd',
```

Looks like one of `author` or `maintainer` is required. Author will be more than just us, `maintainer` sounds a little better to describe the license holder.